src/main/java/com/celements/model/access/ContextExecutor.java

```diff
@@ -27,11 +29,10 @@
   }
 
   public T execute() throws E {
+    checkState(wiki != null, "No wiki set for ContextExecutor");
     WikiReference currWiki = getContext().getWikiRef();
     try {
-      if (wiki != null) {
-        getContext().setWikiRef(wiki);
-      }
+      getContext().setWikiRef(wiki);
```

Comment:
Ja das ist beabsichtigt. Der ContextExecutor ist eine Neuentwicklung und wurde noch nicht in den dev gemergt. Diese Verhaltensänderung wurde von Fabian explizit verlangt: in PR #76